content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigADSCALE.md

```diff
@@ -79,75 +119,37 @@ To check the AD join progress, open the <i class="material-icons" aria-hidden="t
+When the import completes, AD users and groups become available while configuring basic dataset permissions or an ACL with TrueNAS cache enabled (enabled by default).
 
-Joining AD also adds default Kerberos realms and generates a default `AD_MACHINE_ACCOUNT` keytab.
+Joining AD also adds default Kerberos realms and generates a default **AD_MACHINE_ACCOUNT** keytab.
 TrueNAS automatically begins using this default keytab and removes any administrator credentials stored in the TrueNAS configuration file.
 
-### Troubleshooting - Resync the Cache
+### Troubleshooting - Resyncing the Cache
 If the cache becomes out of sync or fewer users than expected are available in the permissions editors, resync it by clicking **Settings** in the **Active Directory** window and selecting **Rebuild Directory Service Cache**.
 
 If you are using Windows Server with 2008 R2 or older, try creating a **Computer** entry on the Windows server Organizational Unit (OU).
 
 When creating the entry, enter the TrueNAS hostname in the name field and make sure it matches the:
 
-* *Hostname*: Go to **Network** and find **Hostname** in the **Global Configuration** window.
-* *NetBIOS alias*: Go to **Credentials > Directory Services** and click **Settings** in the **Active Directory** window. Click **Advanced Options** and find the **NetBIOS alias**.
-
-## Using Shell Commands
-
-You can go to **System Settings > Shell** and enter various commands to get more details about the AD connection and users:
-
-*AD current state*: `midclt call activedirectory.get_state`.
-
-*Connected LDAP server details*: `midclt call activedirectory.domain_info | jq`. For example:
-  ```
-  truenas# midclt call activedirectory.domain_info | jq
-  {
-    "LDAP server": "192.168.1.125",
-    "LDAP server name": "DC01.HOMEDOM.FUN",
-    "Realm": "HOMEDOM.FUN",
-    "Bind Path": "dc=HOMEDOM,dc=FUN",
-    "LDAP port": 389,
-    "Server time": 1593026080,
-    "KDC server": "192.168.1.125",
-    "Server time offset": 5,
-    "Last machine account password change": 1592423446
-  }
-  ```
-
-*View AD users*: `wbinfo -u`.
-* Enter `getent passwd DOMAIN\\<user>` to see more user details (`<user>` = desired user name).
-* If `wbinfo -u` shows more users than are available when configuring permissions and the TrueNAS cache is enabled, go to **Directory Services**, click *Settings* in the *Active Directory* window, and increase the *AD Timeout* value.
-
-*View AD groups*: `wbinfo -g`. Enter `getent group DOMAIN\\domain\ users` to see more details.
-
-*View domains*: `wbinfo -m`.
+* **Hostname**: Go to **Network** and find **Hostname** in the **Global Configuration** window.
+* **NetBIOS alias**: Go to **Credentials > Directory Services** and click **Settings** in the **Active Directory** window. Click **Advanced Options** and find the **NetBIOS alias**.
```

Comment:
What are you doing with nebios aliases here? Netbios aliases should be unique (if they are used at all). The netbios name is what is used to join AD and will be used to update AD DNS (if that default option is enabled)

Reply:
corrected in new edits.